### PR TITLE
refactor: describe experimental being on hold

### PR DESCRIPTION
**Pull request by Sleinmaster**
## Description

Describe the experimental version in .experimental and .versions as being on hold at not receiving updates at the moment.

Files changed:
- experimental.ts
- versions.ts
- weather.ts
- installer.ts
- faq.ts

## Discord Username

slein15

### CODE REVIEW

**Comment by NUTTTTTTTT** [comment]
Dropped you a line on discord regarding the changes to .exp, will do full review following that discussion.

File: src/commands/aircraft/weather.ts [hunk 1]
```diff
@@ -7,15 +7,15 @@ const weatherEmbed = makeEmbed({
     description: makeLines([
         'The A32NX does not have an operating weather radar as it is not currently possible to implement a high-fidelity A320 weather radar or weather map in MSFS.',
         '',
-        'The terrain map (TERR on ND) is currently only available on the development + experimental versions via [SimBridge.](https://docs.flybywiresim.com/simbridge/)',
+        'The terrain map (TERR on ND) is currently only available on the development version via [SimBridge.](https://docs.flybywiresim.com/simbridge/)',
```
Comment (Valastiri) [inline src/commands/aircraft/weather.ts:10]:
```suggestion
        'The terrain map (TERR on ND) is currently only available on the Stable + Development versions via [SimBridge.](https://docs.flybywiresim.com/simbridge/)',
```

File: src/commands/general/installer.ts [hunk 2]
```diff
@@ -4,13 +4,13 @@ import { makeEmbed } from '../../lib/embed';
 
 const installerEmbed = makeEmbed({
     title: 'Installer',
-    description: 'Download the new A32NX Installer where you can select either the Stable, Developer or an Experimental Version, and download and install the addon directly into your Community Folder, [download here](https://api.flybywiresim.com/installer)',
+    description: 'Download the A32NX Installer where you can select either the Stable or Developer Version, and download and install the addon directly into your Community Folder, [download here](https://api.flybywiresim.com/installer)',
```
Comment (Valastiri) [inline src/commands/general/installer.ts:7]:
```suggestion
    description: 'Download the A32NX Installer where you can select either the Stable or Development Version, and download and install the addon directly into your Community Folder, [download here](https://api.flybywiresim.com/installer)',
```

File: src/commands/moderation/faq.ts [hunk 3]
```diff
@@ -51,7 +51,7 @@ const faqEmbeds = [
             },
             {
                 name: 'Why is my version not the same as what I see others using?',
-                value: `> We have three versions, Stable, Development (Master) and Experimental. The Stable version is a 'snapshot' of the development which we regard as stable with the current version of the simulator. The Development build is updated daily and is a constant work in progress and although we test each update thoroughly, minor issues may occur from time to time. If you find this to be the case, you can report these issues in <#${Channels.A32NX_SUPPORT}>.`,
+                value: `> We have three versions, Stable and Development (Master). The Stable version is a 'snapshot' of the development which we regard as stable with the current version of the simulator. The Development build is updated daily and is a constant work in progress and although we test each update thoroughly, minor issues may occur from time to time. If you find this to be the case, you can report these issues in <#${Channels.A32NX_SUPPORT}>.`,
```
Comment (Valastiri) [inline src/commands/moderation/faq.ts:54]:
```suggestion
                value: `> We have two versions, Stable and Development (Master). The Stable version is a 'snapshot' of the development which we regard as stable with the current version of the simulator. The Development build is updated daily and is a constant work in progress and although we test each update thoroughly, minor issues may occur from time to time. If you find this to be the case, you can report these issues in <#${Channels.A32NX_SUPPORT}>.`,
```

**Review by Valastiri — CHANGES_REQUESTED** [review]
corrections

File: src/commands/aircraft/experimental.ts [hunk 4]
```diff
@@ -1,16 +1,13 @@
...
+        'Our Experimental version is temporarily on hold and is not being updated. All of its features have been moved to the Development Version. ',
         '',
-        'Please use the appropriate discord channel or forum to discuss any issues:',
-        `<#${Channels.EXP_CFMS_ISSUES}>`,
-        '',
-        'The Experimental version is a test version to find problems, issues and to improve functionality based on your feedback. It is not meant to be used for daily use or serious flights with an Online ATC service.',
+        'The function of the Experimental version is to test high-impact features which aren't ready for the wider public yet. As there are currently no such features being tested and it being easier to introduce new features into the Development version and maintain it, it was decided to put the Experimental version on hold.',
```
Comment (Sleinmaster) [inline src/commands/aircraft/experimental.ts:10]:
```suggestion
        'The function of the Experimental version is to test high-impact features which aren\'t ready for the wider public yet. As there are currently no such features being tested and it being easier to introduce new features into the Development version and maintain it, it was decided to put the Experimental version on hold.',
```

File: src/commands/aircraft/weather.ts (hunk 1 above)
Comment (Sleinmaster) [inline src/commands/aircraft/weather.ts:10]:
```suggestion
        'The terrain map (TERR on ND) is available on all versions via [SimBridge.](https://docs.flybywiresim.com/simbridge/)',
```

**Review by Valastiri — APPROVED** [review]

**Comment by NUTTTTTTTT** [comment]
Great changes, thanks for this!

**Pull request merged**